.github/workflows/release.yml

```diff
@@ -113,17 +111,19 @@ jobs:
       - name: Poetry update lock and install virtual environment
         # Necessary both to update the package dependencies and account for the new tag
         run: make update-install
-      - name: Package Docs
+      - name: Package and Publish Docs
         # We run this step before the package build, so that the docs are packaged before code obfuscation as no
         # references are generated for obfuscated/compiled code.
-        uses: causalens/docs-upload-action@v0.0.2
-        with:
-          artifactory-password: ${{ secrets.JF_PASSWORD }}
-          artifactory-username: ${{ secrets.JF_USERNAME }}
-          artifactory-url: https://causalens.jfrog.io/artifactory
-          package-name: cai-causal-graph
-          package-paths: cai_causal_graph
-          version: ${{ steps.extract_tag.outputs.new_version }}
+        env:
+          ARTIFACTORY_URL: https://causalens.jfrog.io/artifactory
+          ARTIFACTORY_USERNAME: ${{ secrets.JF_USERNAME }}
+          ARTIFACTORY_PASSWORD: ${{ secrets.JF_PASSWORD }}
```

Comment:
@sam-causalens just need these secrets added to the repo.

Reply:
We also need `GH_ACTIONS_KEY`

Reply:
I've added the jFrog ones for you, but I don't feel comfortable adding the GH_ACTIONS_KEY. I have check and can't seem to create one that is scoped to just this repo so I don't want to risk that in a public repository. I think you'll have to tweak your build process a bit to line up with the Dara Teams process where they push a PR with the version bump and changelog update rather than having it work automatically.

Reply:
Discussed offline, actioned, and tested.